skate-plugin/project-gen/src/jvmMain/kotlin/slack/tooling/aibot/ChatColors.kt

```diff
@@ -15,21 +15,18 @@
  */
 package slack.tooling.aibot
 
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.ReadOnlyComposable
 import androidx.compose.ui.graphics.Color
-import org.jetbrains.jewel.foundation.theme.JewelTheme
 
 object ChatColors {
   val promptBackground = Color(0xFF45494A)
 
   // Color(0xFF2d2f30) responseBackground
-  val responseBackground: Color
-    @Composable @ReadOnlyComposable get() = JewelTheme.globalColors.infoContent
+  val responseBackground = Color(0xFF2d2f30)
+  //    @Composable @ReadOnlyComposable get() = JewelTheme.globalColors.infoContent
```

Comment:
?

Reply:
I reverted the chat colors back to the original because I didn't like how the Jewel Theme muted the colors 

This is the Jewel color:
<img width="154" alt="Screenshot 2024-09-10 at 1 40 34 PM" src="https://github.com/user-attachments/assets/45794699-d54a-4a1d-90ae-33539ce2b426">

This is the original:
<img width="127" alt="Screenshot 2024-09-10 at 1 41 01 PM" src="https://github.com/user-attachments/assets/b59de6e1-eec6-46ed-9acc-aef6de350512">

I just commented out the logic in case we wanted to bring it back but I can delete these lines altogether.

Reply:
Let's save that for another PR since it's sort of orthogonal to this one